src/commands/diff.ts

```diff
@@ -26,7 +26,7 @@ export default class Diff extends Command {
       char: 'f',
       description: 'format of the output',
       default: 'json',
```

Comment:
Should we have `yaml` as default?

Reply:
I'm not sure. We can use yaml instead, but not sure what benefit we are getting for it.

Reply:
Just telling because I see most people use yaml format so keeping it as default could be one less flag as input. I think we should see what most people prefer.

Reply:
Good idea. Perhaps I should drop in a message in slack asking which format they prefer :thinking:

Reply:
Yeah, maybe a small poll about AsyncAPI tools would be benefit us. I think we should have surveys about AsyncAPI, do we have anything like that?

Reply:
I don't think we have any survey platform :thinking:

Reply:
YAML is more preferable if you write AsyncAPI document for humans (as documentation), but if you perform some action then JSON is more preferable. We can make survey on the slack but in my opinion `yaml` should be treated as default. We can also make it in this way that first check input format (json or yaml) and then perform action based on that format.

Reply:
However it's a output from tool to describe changes, so `json` is good for me as default option 👍🏼